README.md

```diff
@@ -34,21 +56,106 @@ Create a json config file with `branch-config` and/or `wctflags` keys:
 `tattoo_config.json`:
 ```
 {
-  "branch-config": {
-    "polymer" : "Polymer/polymer#lazy-register-extends",
-    "paper-button" : "PolymerElements/paper-button#master"
-  },
-  "wctflags": ["-b", "canary"]
+  "test": [
+    "PolymerElements/paper-button#2.0-preview"
+  ],
+  "wctflags": ["--local", "canary"]
 }
-
 ```
+
 Then run:
+
 ```
 tattoo
 ```
 
+Config files support most of the same options as the command-line flags:
 
-***Fix broken installation***
+* `"test": ["PolymerElements/paper-button#2.0-preview", "PolymerElements/*", etc]`
```

Comment:
Right now the system just throws up if there are multiple repo references with the same name before it initiates any cloning or bower installation etc.  This is true even if there are two identical/duplicate repo references.  It should be smarter about how it does that.  We discussed in person about the possibility in the future of supporting exploding a matrix of branches etc, but this is thorny and so the best thing to do is disallow it.

However, I think multiple references to the same repository should probably just favor last-in, which would make it convenient to say that the following implies test all the paper elements on the 2.0 preview branches, except paper-hat should be tested on its special branch.

```
tattoo PolymerElements/paper-*#2.0-preview PolymerElements/paper-hat#special-branch
```

Then tattoo would only need to throw/exit when a package with the same name is referenced from two separate owners/orgs.  Like this wouldn't work:
```
tattoo Hooli/cool-thing PiedPiper/cool-thing
```

And, by extension, the following would only fail *if* the owners had a `cool-` element name in common.
```
tattoo Hooli/cool-* PiedPiper/cool-*
```